ios/SalesforceReact/SFNetReactBridge.m

```diff
@@ -100,9 +101,15 @@ @implementation SFNetReactBridge
     }
     SFRestAPI *restApiInstance = doesNotRequireAuthentication ? [SFRestAPI sharedGlobalInstance] : [SFRestAPI sharedInstance];
 
+    __weak typeof(self) weakSelf = self;
     [restApiInstance sendRequest:request
                                       failureBlock:^(id response, NSError *e, NSURLResponse *rawResponse) {
-                                          callback(@[RCTMakeError(@"sendRequest failed", e, nil)]);
+                                          __strong typeof(self) strongSelf = weakSelf;
+                                          NSMutableDictionary *errorDictionary = [NSMutableDictionary new];
+                                          errorDictionary[@"error"] = e.localizedDescription;
+                                          errorDictionary[@"urlResponse"] = [rawResponse asDictionary];
+                                          errorDictionary[@"response"] = [strongSelf serializableResponse:response rawResponse:rawResponse];
+                                          callback(@[RCTMakeError(@"sendRequest failed", nil, errorDictionary)]);
```

Comment:
RCTMakeError makes "sendRequest failed" a fourth entry in the error dictionary under "message", I made the rest the same as hybrid